cirq/ops/common_gates.py

```diff
@@ -115,6 +115,29 @@ def _trace_distance_bound_(self) -> Optional[float]:
             return None
         return abs(np.sin(self._exponent * 0.5 * np.pi))
 
+    def controlled(self,
+                   num_controls: int = None,
+                   control_values: Optional[Sequence[
+                       Union[int, Collection[int]]]] = None,
+                   control_qid_shape: Optional[Tuple[int, ...]] = None
+                  ) -> raw_types.Gate:
+        """
+        Specialize controlled for XPow to return corresponding controlled
+        CNotPow when the last control (which acts first semantically) is a
+        default-type control qubit.
```

Comment:
I would actually revise this whole pydoc to make it more clear.  It's fairly confusing.

Reply:
Done